examples/create-react-app-material-ui/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "create-react-app-material-ui",
```

Comment:
My though is the following, what's the point of having the material-ui suffix when the example is already scoped to the repository?